src/views/SearchRedesign/SearchResults.tsx

```diff
@@ -50,6 +54,18 @@ export const SearchResults: FunctionComponent = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [results, offset, pageLimit]);
 
+  // Reset offset and update url when filters & sort change
+  // We want to avoid doing this on first render / when a user directly navigates to a search URL
+  // so we keep a ref to prevent this
```

Comment:
Are all of the filters always off when a user first navigates to the search results page / is that the assumption being made here? Is this being done as an optimization or to prevent some kind of weird UI behavior?

Reply:
So the main functionality of this side effect is to effectively reset the search when any of the filters or sort change. I.E, if I'm on the 4th page of results and change my sort or filter by a different language set, I should go back to the first page.

The check on the first render is to prevent the scenario where a user navigates directly to a url such as `/search?q=foo&offset=3` and the offset would be reset. I don't imagine users sharing search result URLs as a common scenario, however, we want to avoid breaking that experience